docs/advanced_initialization.md

```diff
@@ -2,6 +2,8 @@
 
 This guide demonstrates advanced initialization patterns for the Skyflow Node SDK, including multiple vault configurations and different credential types.
 
+Use multiple vault configurations when your application needs to access data across different Skyflow vaults, such as separating production and staging data, or managing data across different geographic regions.
```

Comment:
edit: remove `separating production and staging data, or`

I don't think we should recommend that customers connect to both staging and prod from the same running app. The other example is solid though.

Reply:
Fair. Removed example.